frontend/src/components/account/Profile.jsx

```diff
@@ -41,7 +41,11 @@ const Profile = function Profile(props) {
   const [confirmed, setConfirmed] = useState(false);
   const [sendConfimationMsg, setSendConfimationMsg] = useState("");
 
-  const currentEvent = "spring_2021"
+  const currentEvent = "Fall 2021"; // fix next year
+  const rsvpStatus = "RSVPed! You're all set; you can also cancel your RSVP anytime.";
+  const acceptedStatus = "You have been accepted to HopHacks. Please RSVP if you plan on participating!"
+  const appCompleteStatus = "Application complete!"
```

Comment:
if we wanna make a better work on refactoring, we could have a separate file like "constant.js" for all the hardcode information, https://stackoverflow.com/questions/39036457/react-create-constants-file

Reply:
This would work, will implement later